src/test/java/com/jorgepatrick/NumberConverterTest.java

```diff
@@ -1,19 +1,32 @@
 package com.jorgepatrick;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.stubbing.OngoingStubbing;
 
 import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
 
 public class NumberConverterTest {
```

Comment:
I think it didn't work because this annotation is missing: `@ExtendWith(MockitoExtension.class)`

Reply:
This annotation will run the test with the Mockito context. You can try and debug and see the values of `RomanNumberValidator ` with and without `@ExtendWith(MockitoExtension.class)` to understand the difference.